### ISSUE TITLE
Windows installer for 2.6 is actually 2.5

### ISSUE THREAD

**Issue by LaaZa** [luspi/photoqt#14]
I wasn't sure if I had updated to 2.6 on windows yet and checked the about dialog which said 2.5. So I downloaded and installed 2.6 installer from the website [photoqt.org](https://photoqt.org/downpopupwindows).  However the about dialog still said 2.5, tested again in sandbox and same result, so it is not a leftover or failure to update. Upon closer inspection the features seem to be that of 2.5 with 2.6 features missing. The installer is labeled 2.6. Linux had proper version and features.

TL;DR Windows installer on photoqt.org appears to be 2.5 instead of claimed 2.6 version.

**Comment by luspi** [comment]
Thanks for letting me know, @LaaZa! I must have gotten mixed up moving the files around somewhere. This is now fixed, the installer should now be really for v2.6.
